scripts/build.js

```diff
@@ -100,6 +100,14 @@ async function generateCalendar() {
       fs.mkdirSync(outputDir, { recursive: true });
     }
 
+    // Create a new JSON file with an array of months that have events
+    const eventsFileOutputPath = path.join(outputDir, "events.js");
```

Comment:
Let's change the name of the file to be more clear. Maybe `months.js` or `dates.js`? Similar the variable name can be changed to better reflect what's in the file.